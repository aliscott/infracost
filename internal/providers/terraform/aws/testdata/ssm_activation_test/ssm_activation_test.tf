provider "aws" {
  region                      = "us-east-1" # <<<<< Try changing this to eu-west-1 to compare the costs
  skip_credentials_validation = true
  skip_requesting_account_id  = true
  access_key                  = "mock_access_key"
  secret_key                  = "mock_secret_key"
}

resource "aws_ssm_activation" "ssm_activation" {
  name = "test_ssm_advanced_activation"
  description = "Test"
  iam_role = "my-test-iam-role"
  registration_limit = 1001
}

resource "aws_ssm_activation" "ssm_activation_withUsage" {
  name = "test_ssm_advanced_activation"
  description = "Test"
  iam_role = "my-test-iam-role"
}
