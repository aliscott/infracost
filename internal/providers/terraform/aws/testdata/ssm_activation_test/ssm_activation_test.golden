
 Name                                           Monthly Qty  Unit            Monthly Cost 
                                                                                          
 aws_ssm_activation.ssm_activation                                                        
 └─ On-prem managed instances (advanced)      Cost depends on usage: $0.00695 per hours   
                                                                                          
 aws_ssm_activation.ssm_activation_withUsage                                              
 └─ On-prem managed instances (advanced)             73,000  hours                $507.35 
                                                                                          
 PROJECT TOTAL                                                                    $507.35 

----------------------------------
To estimate usage-based resources use --usage-file, see https://infracost.io/usage-file